internal/storage/postgres/region/get_test.go

```diff
@@ -0,0 +1,328 @@
+package region
+
+import (
+	"context"
+	"testing"
+
+	"github.com/green-ecolution/green-ecolution-backend/internal/entities"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRegionRepository_GetAll(t *testing.T) {
+	t.Run("Get all regions should return all regions ordered by id", func(t *testing.T) {
```

Comment:
We should use unified naming of tests. 

I always did something like:
`should return all regions ordered by id`,
`should return empty region list on empty database` …

I do not have a preference what is better.